### DISCUSSION TITLE
[Feature Request] Add option to revoke token based on secret

### DISCUSSION THREAD

**Discussion by WaaaghNL** [codeigniter4/shield#835] [Ideas]
Hi,

is it a good idea to add the option to revoke the access token based on the secret code thats saved to the database. At this moment it's possible to remove the AccessToken if you know the raw version of it. If it's lost you can only remove it by doing a database call to delete it. 

I was thinking about coping the revokeAccessToken($token) function and remove the hash function from it. Thats how made a POC and that works but i dont have a good name for the function yet. Any ideas?

**Comment by kenjis** [comment]
See #838
